packages/mdc-auto-init/index.js

```diff
@@ -41,14 +42,18 @@ export default function mdcAutoInit(root = document, warn = CONSOLE_WARN) {
     }
 
     // TODO: Should we make an eslint rule for an attachTo() static method?
-    const component = Ctor.attachTo(node);
+    component = Ctor.attachTo(node);
     Object.defineProperty(node, ctorName, {
       value: component,
       writable: false,
       enumerable: false,
       configurable: true,
     });
   }
+
+  if (component !== null) {
```

Comment:
If I understand correctly the original issue was regarding having a way to tell when _all_ components initialize, so I agree that should be the main goal of this PR.

Having one event fire per component init seems potentially excessive and is perhaps a separate bridge we should cross if a convincing use case arises?

Also, I'm not sure whether it makes sense to have the "all initialized" event emit from a particular component, since which component emits it could change just based on how an app changes during development.  I'm open to thoughts/arguments otherwise here though.